org.gazebosim.Gazebo.yaml

```
@@ -0,0 +1,675 @@
+app-id: org.gazebosim.Gazebo
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+command: gazebo
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
```

```suggestion
  - --socket=x11
```

It requires X11 and doesn't want wayland.

---

Thanks,
I found that glew does not support Wayland yet. So anything that use glew must disable wayland.